ch_backup/logic/lock_manager.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, lock_conf: dict, zk_ctl: Optional[ZookeeperCTL]) -> None:
         self._exitcode = lock_conf.get("exitcode")
         self._distributed = zk_ctl is None
         self._disabled = False
+        self._lock_timeout = lock_conf.get("lock_timeout")
```

Comment:
https://github.com/yandex/ch-backup/actions/runs/6326642673/job/17181029960?pr=73
You can add `too-many-instance-attributes` to the 21'st line to get rid of pylint error